server/routes/index.js

```diff
@@ -1,8 +1,120 @@
 const express = require("express");
 const router = express.Router();
+const bcrypt = require("bcrypt");
+const jwt = require("jsonwebtoken");
+const auth = require("../middleware/auth");
+const User = require("../models/user");
 
-router.get("/welcome", function (req, res, next) {
+// testing root route
+router.get("/", function (req, res, next) {
   res.status(200).send({ welcomeMessage: "Step 1 (completed)" });
 });
 
+// register route
+router.post("/register", async (req, res) => {
```

Comment:
Can we make all back-end routes start with API? We can do that when we import the routes in app.js